### ISSUE TITLE
Can't install `py2neo`

### ISSUE THREAD

**Issue by ShutdownRepo** [Dramelac/GoldenCopy#1]
```
# pipx install goldencopy
Fatal error from pip prevented installation. Full pip output in file:
    /root/.local/pipx/logs/cmd_2023-10-14_18.58.30_pip_errors.log

pip seemed to fail to build package:
    goldencopy

Some possibly relevant errors from pip install:
    ERROR: Cannot install goldencopy==1.1, goldencopy==1.3.3 and goldencopy==1.3.4 because these package versions have conflicting dependencies.
    ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts

Error installing goldencopy.
```
Because `py2neo` cannot be installed anymore. See https://community.neo4j.com/t/farewell-py2neo-what-happens-now/64419.

**Comment by ShutdownRepo** [comment]
Temporary fix below:
```bash
git clone https://github.com/Dramelac/GoldenCopy
cd GoldenCopy
python3 -m venv ./venv/
source ./venv/bin/activate
pip3 install --no-deps .
pip3 install git+https://github.com/elena/py2neo
```

**Comment by Dramelac** [comment]
Quick fix applied in 1.3.5 (from source only for now)
